modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/common/RESTTestBase.java

```diff
@@ -321,6 +321,21 @@ protected Response getResponseOfGetWithQueryParams(String endpointUri, Map<Strin
                 .get(endpointUri);
     }
 
+    /**
+     * Invoke given endpointURL for GET with OAuth2 authentication, using the provided token.
+     *
+     * @param endpointURL  endpoint to be invoked
+     * @param accessToken  OAuth2 access token
+     * @return response
```

Comment:
```suggestion
     * @param endpointURL Endpoint to be invoked.
     * @param accessToken OAuth2 access token.
     * @return Response.
```

Reply:
Fix formatting of javadoc comments in other places as well.

Reply:
This is updated in all places.